evidence/model/graphql.go

```diff
@@ -0,0 +1,26 @@
+package model
+
```

Comment:
Sounds like `metadata.go` and `MetadataResponse` / `PackageVersionsResponse`.

Reply:
done